README.md

```diff
@@ -8,45 +8,56 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 ### Mission
 
-The Beman Project’s [mission](docs/missionstatement.md) is to **support the efficient design and adoption of the highest quality C++ standard libraries** through implementation experience, user feedback, and technical expertise.
+The Beman Project’s [mission](docs/missionstatement.md) is to **support the
```

Comment:
interesting -- I assume the linter is complaining about the dashes here?

Reply:
Yeah it's designed to enforce a consistent listing symbol, and the default is `*`, I think I could switch it to `-`